lib/patches/db/pg.rb

```diff
@@ -92,11 +92,29 @@ def async_exec(*args,&blk)
     start        = Time.now
     result       = exec_without_profiling(*args,&blk)
     elapsed_time = SqlPatches.elapsed_time(start)
-    record       = ::Rack::MiniProfiler.record_sql(args[0], elapsed_time)
+    record       = ::Rack::MiniProfiler.record_sql(args[0], elapsed_time, get_binds(*args))
     result.instance_variable_set("@miniprofiler_sql_id", record) if result
 
     result
   end
-
+  
+  def get_binds(*args)
+    return if args[1].nil?
```

Comment:
also lots of regexes here which should be constants, I would also do a while loop here cause perf is super sensitive for profilers